src/app/eventdashboard/page.tsx

```diff
@@ -0,0 +1,10 @@
+import EventDashboardItem from '@/components/cards/event-dashboard-item';
+
+export default function EventDashboard() {
+  return (
+    <main className="flex min-h-screen flex-col items-center justify-between gap-y-24">
```

Comment:
```suggestion
    <main className="flex min-h-screen flex-col items-center justify-between gap-y-24 my-12">
```
Für Test gerne sowas, aber juckt